### PR TITLE
doc: Add Docker on Windows to known issues.

### LINKED ISSUE #838: Error Creating Cluster on Windows
**What happened**:
Running `kind create cluster` (in PowerShell) gives an error:

>PS C:\kind> kind create cluster
Creating cluster "kind" ...
 ✓ Ensuring node image (kindest/node:v1.15.3) 🖼
[31mERRO[0m[14:30:39] docker: Error response from daemon: Windows does not support privileged mode.
[31mERRO[0m[14:30:39] See 'docker run --help'.
 ✗ Preparing nodes 📦
[31mERRO[0m[14:30:39] docker run error: exit status 125
Error: failed to create cluster: docker run error: exit status 125

**What you expected to happen**:
For the cluster to be created (like shown on the home screen).

**How to reproduce it (as minimally and precisely as possible)**:
* Go to https://github.com/kubernetes-sigs/kind/releases
* For Release 0.5.1, Find the "► Assets" dropdown and then the Windows download.
* Download the file.
* Move the downloaded file to `c:\kind`
* Rename the downloaded file to kind.exe
* Added `c:\kind` to my PATH
* Opened PowerShell and ran `kind create cluster`

Got the following error:
>Error: failed to create cluster: docker run error: exit status 125

**Anything else we need to know?**:
No

**Environment:**

- kind version: (use `kind version`): v0.5.1
- Kubernetes version: (use `kubectl version`): 
-- Client: 1.14 (Platform: windows/amd64)
-- Server: 1.15 (Platform: linux/amd64)
- Docker version: (use `docker info`):
-- Server 19.03.1
-- Kernel 10.0 18362
- OS (e.g. from `/etc/os-release`): Windows 10 Enterprise Version 1903

### PR DESCRIPTION
**Pull request by tao12345666333**
Maybe fix #838 

xref: #814 

/cc @BenTheElder @Vaccano @vrock28

### CODE REVIEW

**Comment by Vaccano** [comment]
Looking at the change, I would suggest adding something indicating that running windows containers is not current feasible.  Maybe even link to the issue where this was discussed.

I think the way it is written does say that kind does not support windows containers, but I think you will get a lot of issues created asking for windows support.  (Just my $0.02.)

File: site/content/docs/user/known-issues.md [hunk 1]
```diff
@@ -248,6 +249,17 @@ open /home/user/.docker/config.json: permission denied
...
+
+## Docker on Windows
+
+[Docker Desktop for Windows][docker desktop for windows] supports running both Linux (the default) and Windows Docker containers.
+
+`kind` will fail create a cluster, if your Docker Desktop for Windows using `Windows containers` mode.
+
+This is because `kind` only distributed docker images base on Linux.
+
+To fix this problem, you can follow the docker's docs [switch between Windows and Linux containers][switch between windows and linux containers].
```
Comment (neolit123) [inline site/content/docs/user/known-issues.md]: this is by design. should this really be in "known issues"?

i think the whole section can be summarized to:

```
## Docker on Windows

[Docker Desktop for Windows][docker-desktop] supports running both Linux (the default) and Windows Docker containers.

kind for Windows requires Linux containers. To switch between Linux and Windows containers see [this page][docker-desktop-switch].
```

Comment (Vaccano) [reply site/content/docs/user/known-issues.md]: I think that summary is perfect!

Comment (tao12345666333) [reply site/content/docs/user/known-issues.md]: Great!

File: site/content/docs/user/known-issues.md [hunk 2]
```diff
@@ -248,6 +249,17 @@ open /home/user/.docker/config.json: permission denied
 To fix this problem, either follow the docker's docs [manage docker as a non root user][manage docker as a non root user],
 or try to use `sudo` before your commands (if you get `command not found` please check [this comment about sudo with kind][sudo with kind]).
 
+
+## Docker on Windows
+
+[Docker Desktop for Windows][docker desktop for windows] supports running both Linux (the default) and Windows Docker containers.
+
+`kind` will fail create a cluster, if your Docker Desktop for Windows using `Windows containers` mode.
```
Comment (BenTheElder) [inline site/content/docs/user/known-issues.md]: is using
```suggestion
`kind` will fail create a cluster, if your Docker Desktop for Windows is using `Windows containers` mode.
```

File: site/content/docs/user/known-issues.md [hunk 3]
```diff
@@ -248,6 +249,17 @@ open /home/user/.docker/config.json: permission denied
...
+
+This is because `kind` only distributed docker images base on Linux.
```
Comment (BenTheElder) [inline site/content/docs/user/known-issues.md]: Only supports linux "node" containers while Windows, Mac, and Linux _host_ machines are supported. Windows node support is not currently feasible due to how windows containers work.

File: site/content/docs/user/known-issues.md (hunk 1 above)
Comment (BenTheElder) [reply site/content/docs/user/known-issues.md]: +1

File: site/content/docs/user/known-issues.md (hunk 2 above)
Comment (tao12345666333) [reply site/content/docs/user/known-issues.md]: Thanks!

File: site/content/docs/user/known-issues.md (hunk 1 above)
Comment (tao12345666333) [reply site/content/docs/user/known-issues.md]: I think this summary is clear :+1:  
I want to make changes based on this suggestions.

**Comment by tao12345666333** [comment]
preview link: https://deploy-preview-840--k8s-kind.netlify.com/docs/user/known-issues/#docker-on-windows

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks!

**Pull request merged**